Core/DISET/private/BaseClient.py

```diff
@@ -574,17 +574,17 @@ def __setKeepAliveLapse( self ):
 
   def _getBaseStub( self ):
     """ Returns a tuple with (self._destinationSrv, newKwargs)
-	self._destinationSrv is what was given as first parameter of the init serviceName
+        self._destinationSrv is what was given as first parameter of the init serviceName
 
-	newKwargs is an updated copy of kwargs:
-	  * kwargs has already been updated by all the initialization functions
-	  * if a delegated DN is not set, but is found in __threadConfig or in the info returned by the protocol
-	    sanitiy check, then we replace it (KW_DELEGATED_DN)
-	  * Same goes for the delegated group KW_DELEGATED_GROUP. In the case of a host,
-	    the value is the same as for VAL_EXTRA_CREDENTIALS_HOST
-	  * if set, we remove the useCertificates (KW_USE_CERTIFICATES) in newKwargs (not sure to know why)
+        newKwargs is an updated copy of kwargs:
+          * kwargs has already been updated by all the initialization functions
+          * if a delegated DN is not set, but is found in __threadConfig or in the info returned by the protocol
+            sanitiy check, then we replace it (KW_DELEGATED_DN)
+          * Same goes for the delegated group KW_DELEGATED_GROUP. In the case of a host,
+            the value is the same as for VAL_EXTRA_CREDENTIALS_HOST
+          * if set, we remove the useCertificates (KW_USE_CERTIFICATES) in newKwargs (not sure to know why)
 
-	This method is just used to return information in case of error in the InnerRPCClient
+        This method is just used to return information in case of error in the InnerRPCClient
```

Comment:
Honestly, I don't know. I recovered this file from backup and copied over integration one.